src/renderer/shared/ui/Typography/common/TextBase.ts

```diff
@@ -3,8 +3,8 @@ import { createElement } from 'react';
 import { cnTw } from '@shared/lib/utils';
 import { TypographyProps } from './types';
 
-const TextBase = ({ as = 'p', align = 'left', className, children }: TypographyProps) => {
+export const TextBase = ({ as = 'p', align = 'left', className, children }: TypographyProps) => {
+  if (!children) return null;
```

Comment:
Instead of doing this
```javascript
{myValue && <FootnoteText>{myValue}</FootnoteText>}
```
can do this with much cleaner code (if `myValue` is falsy it won't render):
```javascript
<FootnoteText>{myValue}</FootnoteText>
```